frontend/src/app/sessions/license-indicator/license-usage.service.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, tap } from 'rxjs';
+import {
+  PublicLicenseServerWithUsage,
+  SettingsModelsourcesT4CLicenseServersService,
+} from '../../openapi';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LicenseUsageWrapperService {
+  private _licenseServerUsage = new BehaviorSubject<
+    PublicLicenseServerWithUsage[] | undefined
+  >(undefined);
+
+  readonly licenseServerUsage$ = this._licenseServerUsage.asObservable();
+
+  constructor(
+    private licenseServerService: SettingsModelsourcesT4CLicenseServersService,
+  ) {
+    this.loadLicenseServerUsage().subscribe();
+  }
+
+  loadLicenseServerUsage(): Observable<PublicLicenseServerWithUsage[]> {
+    return this.licenseServerService
+      .getT4cLicenseServersUsage()
+      .pipe(tap((usage) => this._licenseServerUsage.next(usage)));
+  }
```

Comment:
```suggestion
    return this.licenseServerService
      .getT4cLicenseServersUsage(undefined, undefined, { context: SKIP_ERROR_HANDLING_CONTEXT })
      .pipe(tap((usage) => this._licenseServerUsage.next(usage)));
  }
```

Skip the error handling to avoid toast error messages when the license server is not reachable.